### DISCUSSION TITLE
Link/Integration to Kdenlive or other video editor

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#49] [Ideas]
![image](https://github.com/friction2d/friction/assets/357488/a5ba53c6-088b-4bc1-80e0-142743fddca3)

Glaxnimate is similar to friction, and have an useful function with kdenlive that links in real time the files.

**Comment by rodlie** [comment]
As far as I know glaxnimate provides an python/MLT integration module/addon for MLT-based video editors. This is not something Friction can provide.

Friction is a standalone application, I can however provide an D-Bus interface/api if wanted.

The current workflow against an NLE should be to export video as ProRes 444 (Alpha), then import that into the video editor. The video editor should detect changes and reload the asset.

**Reply by Matthi-exe** [reply]
hey dodlie cant you pls provides a documentation on using the api for friction integration in kdenlive

**Reply by rodlie** [reply]
API is on the roadmap for v1.1, sometime this year (maybe). API goes both way, communication will need to happen between Friction and Kdenlive, and in OSS it's hard to agree on a common way to communicate (app to app).

At the moment this is not a priority (unless this is something Kdenlive wants this year).
